modules/openapi-generator/src/main/resources/python-nextgen/api.mustache

```diff
@@ -0,0 +1,264 @@
+                raise ApiTypeError(
+                    "Got an unexpected keyword argument '%s'"
+                    " to method {{operationId}}" % key
+                )
+            _params[_key] = _val
+        del _params['kwargs']
+
+        _collection_formats = {}
+
+        # process the path parameters
+        _path_params = {}
+{{#pathParams}}
+        if _params['{{paramName}}']:
+            _path_params['{{baseName}}'] = _params['{{paramName}}']
+            {{#isArray}}
+            _collection_formats['{{baseName}}'] = '{{collectionFormat}}'
+            {{/isArray}}
+{{/pathParams}}
+
+        # process the query parameters
+        _query_params = []
+{{#queryParams}}
+        if _params.get('{{paramName}}') is not None:  # noqa: E501
+            _query_params.append(('{{baseName}}', _params['{{paramName}}'])){{#isArray}}
+            _collection_formats['{{baseName}}'] = '{{collectionFormat}}'{{/isArray}}
+{{/queryParams}}
+
+        # process the header parameters
+        _header_params = dict(_params.get('_headers', {}))
+{{#headerParams}}
+        if _params['{{paramName}}']:
+            _header_params['{{baseName}}'] = _params['{{paramName}}']{{#isArray}}
+            _collection_formats['{{baseName}}'] = '{{collectionFormat}}'{{/isArray}}
+{{/headerParams}}
+
+        # process the form parameters
+        _form_params = []
+        _files = {}
+{{#formParams}}
+        if _params['{{paramName}}']:
+            {{^isFile}}
+            _form_params.append(('{{baseName}}', _params['{{paramName}}']))
+            {{/isFile}}
+            {{#isFile}}
+            _files['{{baseName}}'] = _params['{{paramName}}']
+            {{/isFile}}
+            {{#isArray}}
+            _collection_formats['{{baseName}}'] = '{{collectionFormat}}'
+            {{/isArray}}
+{{/formParams}}
+
+        # process the body parameter
+        _body_params = None
+{{#bodyParam}}
+        if _params['{{paramName}}']:
+            _body_params = _params['{{paramName}}']
+{{/bodyParam}}
+
+        {{#hasProduces}}
+        # set the HTTP header `Accept`
+        _header_params['Accept'] = self.api_client.select_header_accept(
+            [{{#produces}}'{{{mediaType}}}'{{^-last}}, {{/-last}}{{/produces}}])  # noqa: E501
+
+        {{/hasProduces}}
+        {{#hasConsumes}}
+        # set the HTTP header `Content-Type`
+        _content_types_list = _params.get('_content_type',
+            self.api_client.select_header_content_type(
+                [{{#consumes}}'{{{mediaType}}}'{{^-last}}, {{/-last}}{{/consumes}}]))
+        if _content_types_list:
+                _header_params['Content-Type'] = _content_types_list
+
+        {{/hasConsumes}}
+        # authentication setting
+        _auth_settings = [{{#authMethods}}'{{name}}'{{^-last}}, {{/-last}}{{/authMethods}}]  # noqa: E501
+
+        {{#returnType}}
+        {{#responses}}
+        {{#-first}}
+        _response_types_map = {
+        {{/-first}}
+            {{^isWildcard}}
```

Comment:
i've used string as the key to avoid compilation error for the time being.

still need to work on better handling of response status code using wild card.

will do it with separate PR after merging this one.